tools/enrollment_preprocessor.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import csv
+from argparse import ArgumentParser
+from dataclasses import dataclass
+from io import BytesIO
+from typing import TextIO
+
+from openpyxl import load_workbook
+from openpyxl.cell import Cell
+
+
+@dataclass
+class User:
+    title: str
+    last_name: str
+    first_name: str
+    email: str
+
+
+@dataclass
+class UserCells:
+    title: Cell | None
+    last_name: Cell
+    first_name: Cell
+    email: Cell
+
+    def value(self) -> User:
+        return User(
+            str(self.title.value) if self.title and self.title.value else "",
+            str(self.last_name.value),
+            str(self.first_name.value),
+            str(self.email.value),
+        )
+
+
+def user_decision(field: str, existing: str, imported: str) -> str:
+    if existing == imported:
+        return existing
+    decision: str = ""
+    while decision not in ("y", "n"):
+        decision = input(f"Do you want to keep the existing user {field}? (y/n) ")
```

Comment:
For me as a first time user of the script, I had to think a bit about what "yes" and "no" imply here, because the question is a bit negated ("Do you want to keep the existing" == "Do you want to not apply the change that is in the data when importing"). Intuitively, I'd expect "yes" to be "Yes, the new data/change is correct" and "no" to mean "No, discard this / please keep the old data". Might be a me-problem though. @janno42 should probably say what the exact prompt should be here.

It might help if we list the value that the user would accept with "y" ->
> Do you want to keep the existing user first_name "Bastius"? (y/n)